lib/workload/stateless/universal-event-archiver/function/achiver.py

```diff
@@ -0,0 +1,43 @@
+import json
+import boto3
+from datetime import datetime, timezone
+import os
+
+# Initialize S3 client
+s3 = boto3.client('s3')
+
+# The name of the bucket
+BUCKET_NAME = os.getenv('BUCKET_NAME')
+
+def handler(event, context):
+    
+    # Current timestamp
+    now = datetime.now(timezone.utc)
+    time_stamp = now.strftime("%H_%M_%S") # for object name
+    time_stamp_details = now.strftime("%Y-%m-%d__%H-%M-%S") # for tagging
+    
+    # Extract the event title (type), reference from the event_schemas example
+    event_title = event.get('detail', {}).get('info', {}).get('title', 'OtherEvents')
```

Comment:
That structure is likely to change. 
Perhaps better to rely on the general Event Bridge event [format](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-events-structure.html)? Particularly the `detail-type` as that should define the event payload structure (`detail`).

Reply:
sure, I am also concern the event data struct. I will apply the defined "detail-type" in each event.